deb/openmediavault/usr/sbin/omv-mkrrdgraph

```diff
@@ -48,11 +48,11 @@ def main():
 			'--daemon', 'unix:{}'.format(openmediavault.getenv(
 				'OMV_RRDCACHED_SOCKETFILE', '/var/run/rrdcached.sock')),
 			'--imgformat', openmediavault.getenv(
-				'OMV_COLLECTD_RRDTOOL_GRAPH_IMGFORMAT', 'PNG'),
+				'OMV_COLLECTD_RRDTOOL_GRAPH_IMGFORMAT', 'SVG'),
 			'--width', openmediavault.getenv(
-				'OMV_COLLECTD_RRDTOOL_GRAPH_WIDTH', '400'),
+				'OMV_COLLECTD_RRDTOOL_GRAPH_WIDTH', '600'),
 			'--height', openmediavault.getenv(
-				'OMV_COLLECTD_RRDTOOL_GRAPH_HEIGHT', '120'),
+				'OMV_COLLECTD_RRDTOOL_GRAPH_HEIGHT', '180'),
```

Comment:
Please use the old size, the big graphs look ugly.

Reply:
Done